share/xtp/xml/excitoncoupling.xml

```diff
@@ -1,25 +1,25 @@
 <options>
 
 <excitoncoupling help="Exciton couplings from serialized orbital files" section="sec:bsecoupling">
-        <classical help="Exciton couplings from classical transition charges" default="0">0</classical>
+        <classical help="Exciton couplings from classical transition charges" default="0" choices="int+">0</classical>
```

Comment:
should be a bool actually. maybe the tag `use_classical`. help=Calculate Couplings from classical charges instead of ab-initio